Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -75,8 +75,6 @@ class MessageContentView: BaseView {
     }
     let showEmotionsButton = EmotionGradientView()
 
-    let newMessageIndicatorView = UIImageView(image: UIImage(systemName: "eyeglasses"))
```

Comment:
Moved this into the footer. While I liked it at the top, I felt it ultimately cluttered up the content, and looked really bad for text+attachments.